data/tribes/scripting/help/building_help.lua

```diff
@@ -511,11 +513,6 @@ function building_help_dependencies_production(tribe, building_description)
 
    -- Produced items
    if (building_description.output_ware_types[1] or building_description.output_worker_types[1]) then
-      result = result .. h3(_"Produces:")
```

Comment:
for consistency I think we should:
1. keep the old code here, having some duplication. 
2. skip the workers part as well and add this to the production section in the same way as for the wares. 
in the latter case we would only have dependencies here which might be good, but we should probably move up the Production section in fron tof this.

Reply:
Don't know if i understand correctly...

Please take a look

Reply:
I took a look:
I like your idea to have the crew required in the production section. And I like how the dependencies are now:   
  
some issues still:
1. I think the new Production section is the most valuable Information and should be put at second place (i.e. in front of the dependencies. 
2.  in the crew required part the experience levels are h2 which should be h3. this has been wrong before this branch but is now more obvious. (maybe we could skip this here as it is in the worker help as well).  
3. the new help does not work for frisians aqua farm, as the resources are spent in a different program then the production happens. 
4. the recruitment part (workers are produced) needs to be shifted to the new Production section, as currently the recruited / produced workers are listed as incoming dependency.

Reply:
> 1. I think the new Production section is the most valuable Information and should be put at second place (i.e. in front of the dependencies.

It depends on the player status i think. For a new player the dependencies are most valuable. Just a guess.

> 2. in the crew required part the experience levels are h2 which should be h3. this has been wrong before this branch but is now more obvious. (maybe we could skip this here as it is in the worker help as well).

We definitely need #3769

> 3. the new help does not work for frisians aqua farm, as the resources are spent in a different program then the production happens.

Added a dummy program like there are for other production sites already. Please check if the values are correct: I think one fish is produced out of two water and one fruit, but i am not sure :-D
 
> 4. the recruitment part (workers are produced) needs to be shifted to the new Production section, as currently the recruited / produced workers are listed as incoming dependency.

Should be fixed

Reply:
No 4 is only half fixed. it is listed now as outgoing dependency but the costs are not displayed in the production section.